charts/apisix/values.yaml

```diff
@@ -312,6 +312,12 @@ logs:
   errorLog: "/dev/stderr"
   errorLogLevel: "warn"
 
+core:
+  worker_rlimit_nofile: 20480
+  events:
+    worker_connections: 10620
+    use: epoll
```

Comment:
> the apisix is a linux system, with `epoll` networker model  better handle `networker connection`

Reply:
ref: [ngx_core_module](http://nginx.org/en/docs/ngx_core_module.html#use) and [events](http://nginx.org/en/docs/events.html) 

```
There is normally no need to specify it explicitly, because nginx will by default use the most efficient method.
```

So is it better to default to nil?

Reply:
yes， i have see, Now that the scene has been clarified, is the display clearer?